public/app/app.bootstrap.js

```diff
@@ -80,6 +80,10 @@
     }).then(function(results) {
       collabosphere.constant('me', results.me.data);
       collabosphere.constant('config', results.config.data);
+      // Support deep linking with generic 'by_id' param passed from other SuiteC tool or email.
+      if (parameters.by_id) {
+        collabosphere.constant('id_parameter', parameters.by_id);
+      }
```

Comment:
@johncrossman I hadn't dug into this `collabosphere.constant` stuff before, but it seems like a clever way to cache the id parameter. Might there be an issue with this parameter hanging around longer than expected and interfering with subsequent navigation? If so perhaps we could explicitly clear it once it's done its job.